docs/Hooks.md

```diff
@@ -121,6 +132,27 @@ fastify.addHook('preHandler', (request, reply, next) => {
 
 *The error will be handled by [`Reply`](https://github.com/fastify/fastify/blob/master/docs/Reply.md#errors).*
 
+#### The `onError` Hook
+
+This hook is useful if you need to do some custom error logging or add some specific header in case of error.<br/>
+It is not intended for changing the error, and calling `reply.send` will throw an exception.<br/>
+This hook will be executed only after the `customErrorHandler` has been executed.<br/>
```

Comment:
Since the `onError` hooks are only run if the custom error handler forwards the error back to Fastify with `reply.send(err)`, I think it would be helpful to explain that here.

Reply:
Done! :)